docs/action-debugging.md

```diff
@@ -1,9 +1,23 @@
 # Debugging
-If the build logs do not provide enough detail on why a build may be failing, some other options exist to assist with troubleshooting.
+If the job logs do not provide enough detail on why a job may be failing, some other options exist to assist with troubleshooting.
+
+## Step Debug Logs
+This is the primary way for customers to debug job failures caused by step failures.
```

Comment:
Apologies, now I'm re-reading my own suggestion, I find that the double use of `failures` read a bit awkwardly.  Replacing `step failures` with `failed steps` makes the sentence sound better to me. .